src/server/node_manager.c

```diff
@@ -5128,6 +5129,7 @@ is_request(int stream, int version)
 			}
 
 			free_attrlist(&reported_hooks);
+			psvrmom->msr_is_hook_checksums_done = 1;
```

Comment:
Yes, then that time do_sync_mom_hookfiles will not set and that flag I am checking.

Regards,
Riyaz